src/Query.php

```diff
@@ -148,59 +160,68 @@ public function withConsistentRead() {
      * Whether to not use consistent read in the query.
      * @return static
      */
-    public function withoutConsistentRead() {
+    public function withoutConsistentRead()
+    {
         $this->consistentRead = false;
         return $this;
     }
-    
+
     /**
      * Whether to return the consumed capacity.
      * @return static
      */
-    public function withConsumedCapacity() {
+    public function withConsumedCapacity()
+    {
         $this->returnConsumedCapacity = true;
         return $this;
     }
-    
+
     /**
      * Whether not to return the consumed capacity.
      * @return static
      */
-    public function withoutConsumedCapacity() {
+    public function withoutConsumedCapacity()
+    {
         $this->returnConsumedCapacity = false;
         return $this;
     }
 
     /**
      * Returns all object that matches the query.
-     * @param Connection $db the dynamodb connection.
-     * @return
+     * @param Connection $db The dynamodb connection.
+     * @return array
      */
-    public function all($db = null) {
+    public function all(Connection $db = null)
+    {
         return $this->createCommand($db)->queryAll();
```

Comment:
Yes, this will call either batchGetItem, scan, or query. I'm still writing the populate method.